docs/tutorials/light-up.md

```diff
@@ -0,0 +1,224 @@
+When you first open the app, you will be prompted to create an account.
+As you do this, you will receive an email with subject line "TP-Link ID: Activation Required" to complete your account registration.
+
+Next, follow the steps in Kasa's [How to set up my TP-Link Smart Plug Switch via Kasa guide](https://www.tp-link.com/us/support/faq/946/) to add your device and connect it to your wifi.
+Once it is connected, you will no longer need to use the mobile app.
+
+Next, open a terminal on your computer and run the following command to download the [smart plug Python API](https://github.com/python-kasa/python-kasa):
+
+```bash
+pip3 install python-kasa
+```
+
+Next, run the following command to return information about your smart device:
+
+```bash
+kasa discover
+```
+
+You should see this command output something like this:
+
+![Terminal output with information about the smart plug including the host, device state (on), timestamp, hardware and software versions, MAC address, location (latitude and longitute), whether the LED is currently on, and the timestamp of when it last turned on. There is also a list of modules (schedule, usage, antitheft, time and cloud).](../img/light-up/kasa-discover-output.png)
+
+Copy the host address (for example, 10.1.11.221).
+You will need to include it in your Python code in a later step.
+
+## Write Python code to control your object detection robot
+
+Navigate to the **CODE SAMPLE** tab on the Viam app.
+Make sure **Python** is selected in the **Language** selector.
+
+Click the **COPY CODE** button.
+Paste the code into a new Python file on your computer.
+This code connects your script to your robot.
+
+Now you are ready to write the code to turn your smart plug on and off based on camera readings.
+You will use a [TFLite](https://www.tensorflow.org/lite) model to detect specific objects, and a corresponding text that holds class labels for your TFLite model.
+For more information on the Vision Service, check out our [documentation](/services/vision/).
+
+In your Python code, as you set up the vision service parameters, you will need to change the `model_path` to where your TFLite package lives, and the `label_path` to where your text file lives.
+You can download them from [here](https://github.com/viam-labs/devrel-demos/tree/main/Light%20up%20bot).
+
+You will also generate a person detector function within your code, to detect a person and trigger the camera.
+
+{{% alert title="Info" color= "info" %}}
+
+Technically, you can detect any object that is listed in the <file>labels.txt</file> (such as a dog or a toilet) but for this tutorial, we are detecting a person.
+
+To detect something else with the camera, just change the string "person" to a different item in the <file>label.txt</file> file.
+
+```python
+               if d.class_name.lower() == "person":
+                   print("This is a person!")
+                   found = True

```

Comment:
I assume you wanted this to be on the same level as in the code sample but that doesn't look good. Generally any editor folks use should help with formatting as well
```suggestion
if d.class_name.lower() == "person":
    print("This is a person!")
    found = True

```